examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -219,6 +222,16 @@ def inference(self, input_batch):
                 prediction = [(token, label_list[prediction]) for token, prediction in zip(tokens, predictions[0].tolist())]
                 inferences.append(prediction)
             logger.info("Model predicted: '%s'", prediction)
+            
+        # Handling inference for text_generation.
+        if self.setup_config["mode"] == "text_generation":
+            
+            for input in input_ids_batch:
+                prompt_length = len(self.tokenizer.decode(input[0]))
+                outputs = self.model.generate(input, max_length=150, do_sample=True, top_p=0.95, top_k=60)
+                generated = self.tokenizer.decode(input) + self.tokenizer.decode(outputs[0])[prompt_length + 1 :]
+            
+                inferences.append(generated)
```

Comment:
I tried to keep the same approach/style as the other transformer examples, so for each text example (prompt) from the batch, there is one instance of generated text (which is put together with the prompt, as in the official example). Max sequence is per every text example. The output will have the length of the max sequence and will be decoded in text. The parameters chosen in this function are the ones used in the official example from Huggingface, because I figured that maybe a lot of people would like see the same example served. Of course this might be improved in the future, by adding more flexibility in parameters selection.

The model size is comparable of the other transformers (~500 mb), but I'm not sure about the speed.

The output is not deterministic, because we can't know in advance what the generated text will be. From the local tests that I've done, each time I got a different output. I think the user can be sure the example is executed correctly when some text is generated, regardless of the content.

Considering this, I'm not sure how I should make assertion tests with this example.